docs/using/config/usage.rst

```diff
@@ -44,10 +44,11 @@ To authenticate with the control service, you will need a copy of the public clu
 
 For ``flocker-deploy``, your API user certificate and key should be in files named ``user.crt`` and ``user.key`` and the cluster certificate in file ``cluster.crt``.
 
-By default, ``flocker-deploy`` will look for these certificate files in the current working directory.
-If this is not where the files are located, you may specify the ``--certificates-directory`` option to ``flocker-deploy``:
+By default, ``flocker-deploy`` will look for these certificate files in the current working directory and expect them to be named :file:`cluster.crt` (the public cluster certificate), :file:`user.crt` (the API user certificate) and :file:`user.key` (the API user's private key).
```

Comment:
You don't need both this paragraph and the previous one. I suggest deleting the previous one.